PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1520,6 +1520,56 @@ def miniAODConfigurationPre(self, process, patMetModuleSequence, postfix):
                              )
             getattr(process,"patCaloMet").addGenMET = False
 
+            ##adding the necessary chs and track met configuration
+            task = getPatAlgosToolsTask(process)
+
+            pfChs = cms.EDFilter("CandPtrSelector", src = cms.InputTag("packedPFCandidates"), cut = cms.string("fromPV(0)>0"))
+            addToProcessAndTask("pfChs", pfChs, process, task)
+            pfMetChs = cms.EDProducer("PFMETProducer",
+                                      src = cms.InputTag('pfChs'),
+                                      alias = cms.string('pfMet'),
+                                      globalThreshold = cms.double(0.0),
+                                      calculateSignificance = cms.bool(False),
+                                      )            
+
+            addToProcessAndTask("pfMetChs", pfMetChs, process, task)
+
+            addMETCollection(process,
+                             labelName = "patChsMet",
+                             metSource = "pfMetChs"
+                             )
+
+            getattr(process,"patChsMet").computeMETSignificance = cms.bool(False)
```

Comment:
here and below - these getattr seem unnecessary. Eg, process.patChsMet does the job and is more readable.

Reply:
this was done so we don't waste time running the significance code since all we need is the central value of the met.

Reply:
hi @zdemirag - i think you answered a different question. I'm purely suggesting a more clear syntax not a functionality change

Eg

process.patChsMet.computeMETSignificant = ....

Reply:
ah sorry, will do it right away.